src/controllers/beneficial.owner.gov.controller.ts

```diff
@@ -118,7 +118,6 @@ const setBeneficialOwnerData = (reqBody: any, id: string): ApplicationDataType =
   data[NonLegalFirmNoc] = (data[NonLegalFirmNoc]) ? [].concat(data[NonLegalFirmNoc]) : [];
 
   data[IsOnSanctionsListKey] = (data[IsOnSanctionsListKey]) ? +data[IsOnSanctionsListKey] : '';
-  data[HasSamePrincipalAddressKey] = (data[HasSamePrincipalAddressKey]) ? +data[HasSamePrincipalAddressKey] : '';
```

Comment:
What line is this duplicating? A line in another class somewhere?

Reply:
In the same function, a few lines above

Reply:
Sorry, now I see it! Thanks.